eng/tox/tox.ini

```diff
@@ -27,13 +27,6 @@ source =
     microsoft
     **/microsoft
 
-
-[base]
-deps =
-  -rdev_requirements.txt
-  {[tools]deps}
-
-
```

Comment:
I am not familiar with tox but it seems that there is no dev_requirements.txt under https://github.com/Azure/azure-sdk-for-python/tree/main/eng/tox